docs/ref/databases.txt

```diff
@@ -367,28 +367,6 @@ this for you (the database backend layer and the model population layer are
 separated internally so the database layer doesn't know it needs to make this
 conversion in this one particular case).
 
-If you're using MySQLdb 1.2.1p2, Django's standard
```

Comment:
I tried to understand the previous paragraph a bit better. I ran: `ALTER TABLE polls_question CHARACTER SET utf8 COLLATE utf8_bin;`, then `type(Question.objects.all()[0].question_text)` but got `str/unicode` on Python 3 and 2, respectively -- so either this note is obsolete or my understanding is lacking. @adamchainz, any MySQL expertise to offer?